server.js

```diff
@@ -66,10 +66,9 @@ try {
 })();
 
 // Use helmet to set security headers
-// disable default HSTS; Ops handles it in stage & prod configs
-app.use(helmet({
-  hsts: false,
-}));
+// only enable HSTS on heroku; Ops handles it in stage & prod configs
+const hsts = AppConstants.NODE_ENV === "heroku" ? true : false;
```

Comment:
`AppConstants.NODE_ENV === "heroku"` is already a Boolean, so no need to re-cast to `? true : false`, but yeah. 👍